internal/server/disk/server.go

```diff
@@ -0,0 +1,78 @@
+package disk
+
+import (
+	"context"
+
+	"github.com/kubernetes-csi/csi-proxy/client/apiversion"
+	"github.com/kubernetes-csi/csi-proxy/internal/server/disk/internal"
+	log "github.com/sirupsen/logrus"
```

Comment:
Do we need to use logrus ? Can't we do with klog ? Although the request tracing is better in logrus there are only few levels of verbosity. Also, since k8s is in the klog, wouldn't it make sense to align ?